dcloud-logs

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Usage:
```

Comment:
Could make this a string and print it with `$1 == --help`

Reply:
sure!